src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {'Posted by '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    <hr />
+    {post.comments.length > 0 ? (
```

Comment:
You are directly checking the length property on post.comments. If for some reason post.comments is undefined, this will result in an error. A safer alternative would be to use conditional chaining (post.comments?.length > 0).